committee/committee.go

```diff
@@ -245,7 +256,12 @@ func (ec *committee) Start(ctx context.Context) (err error) {
 func (ec *committee) Stop(ctx context.Context) error {
 	ec.mutex.Lock()
 	defer ec.mutex.Unlock()
-	ec.terminate <- true
+
+	defer func() { ec.terminated = true }()
+	if ec.terminated {
+		return nil
+	}
+	close(ec.terminate)
```

Comment:
ec.terminated = true here? so no need defer()

Reply:
want set terminated after everything is closed